### ISSUE TITLE
Fragments into inactive tabs don't receive onPause/onResume events.

### ISSUE THREAD

**Issue by marcouberti** [moallemi/MultiNavHost#4]
When switching tabs, the current fragments of previously selected tabs don't receive the onPause()/onResume() events. This can be problematic in scenarios when you have to release resources, like stopping a video player. I checked this Google's sample and the events are triggered as expected.

[Google Navigation Advanced Sample](https://github.com/android/architecture-components-samples/tree/master/NavigationAdvancedSample)

**Comment by amit7127** [comment]
Yes facing the same issue. Visible Fragment's onResume() not getting called. Any way to achieve this?

**Comment by Faiyyazauxy** [comment]
Same issue for me but the google navigation advanced sample has its own issue
